CMSgov/easi-app
# Easi 2969/user account initial implementation

# EASI-2969

## Changes and Description

- Create initial user account implementation
- Extend the User Info properties to include first and last name



## How to test this change
1. Log into the app
2. Look in the db and confirm that an account reference is made for your account 
3. Enable OKTA login and log in with okta
4. Confirm that an account is made to reference your account
5. Test the GQL endpoint by searching for the account by username with the query in the updated postman collection

Test the utility
 1. Test the script by running `go run ./cmd/populate_user_table/*.go`
 2. Select options to run with <kbd>space</kbd>
 3. Run the options by pressing <kbd>enter</kbd>





## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
